R/importNGWMN_wml2.R

```diff
@@ -112,19 +115,38 @@ importNGWMN_wml2 <- function(input, asDateTime=FALSE, tz){
   return(mergedDF)
 }
 
-#' parse the timeseries portion of a waterML2 file
+#' Parse the WaterML2 timeseries portion of a waterML2 file
 #' 
 #' Returns data frame columns of all information with each time series measurement;
 #' Anything defined as a default, is returned as an attribute of that data frame.
 #' 
 #' @param input XML with only the wml2:MeasurementTimeseries node and children
+#' @param asDateTime logical, if \code{TRUE} returns date and time as POSIXct, if \code{FALSE}, character
+#' @param tz character to set timezone attribute of datetime. Default is an empty quote, which converts the 
+#' datetimes to UTC (properly accounting for daylight savings times based on the data's provided time zone offset).
+#' Possible values are "America/New_York","America/Chicago", "America/Denver","America/Los_Angeles",
+#' "America/Anchorage","America/Honolulu","America/Jamaica","America/Managua","America/Phoenix", and "America/Metlakatla"
 #' @importFrom xml2 xml_attr xml_find_all xml_text 
 #' @importFrom dplyr mutate
 #' @importFrom lubridate parse_date_time
 #' @export
-parseWaterML2Timeseries <- function(input, asDateTime, tz) {
-  gmlID <- xml_attr(input,"id") #TODO: make this an attribute
-  TVP <- xml_find_all(input, ".//wml2:MeasurementTVP")#time-value pairs
+#' @examples 
+#' baseURL <- "https://waterservices.usgs.gov/nwis/dv/?format=waterml,2.0"
+#' URL <- paste(baseURL, "sites=01646500",
+#'      "startDT=2014-09-01",
+#'      "endDT=2014-09-08",
+#'      "statCd=00003",
+#'      "parameterCd=00060",sep="&")
+#' \dontrun{
+#' timesereies <- importWaterML2(URL, asDateTime=TRUE, tz="UTC")
+#' } 
+importWaterML2 <- function(input, asDateTime=FALSE, tz="UTC") {
```

Comment:
I left it to be consistent with the other "import" functions. It is possible people use this function as a standalone (the certainly use the other import functions for local files. As WaterML2 becomes even more popular, this function could be used on local files too).